src/main/java/echo/Server.java

```diff
@@ -0,0 +1,91 @@
+package echo;
+
+import java.io.*;
+import java.net.*;
+
+
+public class Server {
+    public static void main(String[] args) throws IOException {
+        runEchoServer();
+    }
+
+    public static ServerSocket serverSocket = null;
+        public static Socket clientSocket = null;
+
+
+
+    public static void runEchoServer() throws IOException {
+
+        try {
+            serverSocket = createServerSocket(7777);
+
+        } catch (IOException e) {
+            errorMessage("Can't listen on port number 7777", e);
+
+        }
+
+        print("Listening for connection...");
+
+        try {
+
```

Comment:
I like the try - catch here too - it's very clear that there are several steps along the way that can fail, all with different messages. It would be clear to the user what went wrong, good job 👏